android/java/res/layout/brave_wallet_reset_content.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+ <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    style="@style/AlertDialogContent">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/brave_wallet_reset_settings_confirmation" />
```

Comment:
that TextView should be set dynamically to replace `Yes` placeholder like that `String.format(
                        getResources().getString(R.string.brave_wallet_reset_settings_confirmation), getResources().getString(R.string.brave_wallet_reset_settings_confirmation_phrase))`